tslint.json

```diff
@@ -0,0 +1,22 @@
+{
+    "defaultSeverity": "error",
+    "extends": [
+        "tslint-react"
+    ],
+    "jsRules": {},
+    "rules": {
+        "member-access": false,
+        "ordered-imports": false,
+        "quotemark": false,
+        "no-console": false,
+        "semicolon": false,
+        "jsx-no-lambda": false
+    },
+    "rulesDirectory": [],
+    "linterOptions": {
+        "exclude": [
+            "config/**/*.js",
+            "node_modules/**/*.ts"
+        ]
+    }
+}
```

Comment:
by the way, instead of creating `tslint.config` i preffer to keep using eslint but extend it to support typescript. 
```
{
  "parser": "@typescript-eslint/parser",
  "parserOptions": {
    "ecmaVersion": 2018,
    "sourceType":  "module",
    "ecmaFeatures": {
      "experimentalObjectRestSpread": true,
      "jsx": true
    }
  },
  "rules": {
    "no-console": "warn",
    "comma-dangle": ["error", "never"],
    "no-unused-vars": ["warn", { "vars": "all", "args": "after-used", "ignoreRestSiblings": false }],
    "import/prefer-default-export": 0,
    "no-useless-constructor": ["warn"],
    "no-trailing-spaces": 0,
    "class-methods-use-this": 0,
    "react/prop-types": 0,
    "arrow-parens": 0,
    "no-restricted-globals": 0,
    "padded-blocks": 0,
    "no-underscore-dangle": 0,
    "no-unused-expressions": 0,
    "no-restricted-syntax": ["warn"],
    "import/no-unresolved": 0,
    "import/named": 2,
    "import/namespace": 2,
    "import/default": 2,
    "import/export": 2,
    "@typescript-eslint/indent": ["error", 2],
    "@typescript-eslint/no-inferrable-types": 0
  },
  "extends": [
    "airbnb-base",
    "plugin:react/recommended",
    "plugin:@typescript-eslint/recommended",
    "plugin:import/errors",
    "plugin:import/warnings"
  ],
  "env": {
    "browser": true,
    "node": true,
    "es6": true
  }
}
```